projects/vdk-heartbeat/src/vdk/internal/heartbeat/hearbeat.py

```diff
@@ -43,6 +43,7 @@ def run(self):
             run_test.setup()
 
             job_controller.enable_deployment()
+            job_controller.wait_job_execution_started()
             job_controller.show_job_details()
 
             run_test.execute_test()
```

Comment:
The reason is, `execute_test` was waiting with configurable max time for the result properties to appear, to conclude the test was successful. There is no need to wait for execution to start in particular -> if results are available, execution had started and completed. 
This change actually introduces RUN_TEST_TIMEOUT_SECONDS * 2

Reply:
The run test instance I am using is EmptyRunTest and that definitely des not do that.
I also looked at a few other instances and they don't seem to do it either. 

Can you point me to the line of code you think is always running?